moveos/moveos-types/src/state.rs

```diff
@@ -91,6 +133,28 @@ pub trait MoveState: MoveType + DeserializeOwned + Serialize {
         MoveValue::simple_deserialize(&blob, &Self::type_layout())
             .expect("Deserialize the MoveValue from MoveState should success")
     }
+
+    fn to_runtime_value(&self) -> Value {
+        let blob = self.to_bytes();
+        Value::simple_deserialize(&blob, &Self::type_layout())
+            .expect("Deserialize the MoveValue from MoveState should success")
+    }
+
+    /// Deserialize the MoveState from MoveRuntime Value
+    fn from_runtime_value(value: Value) -> Result<Self>
+    where
+        Self: Sized,
+    {
+        let blob = value
+            .simple_serialize(&Self::type_layout())
+            .ok_or_else(|| {
+                anyhow::anyhow!(
+                    "Serialize the MoveState from Value error: {:?}",
```

Comment:
Comments and error msg are inconsistent?

Reply:
The AI generated. I will fix it in a new PR.